utils/debug/TileDebug.js

```diff
@@ -44,13 +36,7 @@ export default function createTileDebugUI(datDebugTool, view, layer, debugInstan
     });
 
     // tiles wireframe
-    gui.add(layer, 'wireframe').name('Wireframe').onChange((newValue) => {
-        layer.wireframe = newValue;
-
-        applyToNodeFirstMaterial(view, layer.object3d, layer.id, (material) => {
-            material.wireframe = newValue;
-        });
-    });
+    GeometryDebug.addWireFrameCheckbox(gui, view, layer);
```

Comment:
Yep, good idea

Reply:
I have changed line 9 into createGeometryDebugUI ...

But I still need to add GeometryDebug.addWireFrameCheckbox...
Because wireframe is not in the createGeometryDebugUI, since wireframe doesn't make sense for points or lines.